MusicAppApp.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MusicAppApp.swift
+//  MusicApp
+//
+//  Created by Dawid Nowacki on 03/02/2024.
+//
+
+import SwiftUI
+
+@main
+struct MusicAppApp: App {
+    let persistenceController = PersistenceController.shared
+
+    var body: some Scene {
+        WindowGroup {
+            ContentView(expandSheet: .constant(true), animation: Namespace().wrappedValue)
```

Comment:
when I built the project I noticed runtime warning (this purple one) with 
_Reading a Namespace property outside View.body. This will result in identifiers that never match any other identifier._
So would only init it in parameter 
```suggestion
            ContentView(expandSheet: .constant(true), animation: Namespace())
```